ruby_on_rails/README.md

```diff
@@ -31,9 +31,12 @@ They are not optional, but the rest of the team can start working even if those
 
 1. [GetSentry](getsentry.md)
 1. [NewRelic](newrelic.md)
-1. [Uptime Monitor](uptime.md)
 1. [Gemnasium](gemnasium.md)
 
+Some services should be enabled only if the customer pays for the monitoring package:
```

Comment:
Should New Relic also come under this section? What other services belong here?

Reply:
Gemnasium, NewRelic, Sentry as well